$OpenBSD: patch-configure,v 1.1 2013/01/31 21:20:32 dcoppa Exp $

Support DESTDIR with findlib-install; unbreak systraced builds
(adapted from upstream git 58e15c3a33ee747e3e58a2c9687f22234f0055b6)

--- configure.orig	Thu Aug 23 12:37:48 2012
+++ configure	Thu Jan 31 11:49:11 2013
@@ -568,6 +568,7 @@ DEBUG
 USE_CC
 HAVE_SVGZ
 LABLGLDIR
+FINDLIBDIR
 HAS_NATIVE_DYNLINK
 HAS_DLL_SUPPORT
 THREADS_LIB
@@ -2607,6 +2608,9 @@ fi
 if test "$CAMLP4O" = no ; then
 	as_fn_error $? "Cannot find camlp4o." "$LINENO" 5
 fi
+
+FINDLIBDIR="`ocamlfind printconf destdir | tr -d '\\r'`"
+echo "ocamlfind library path is $FINDLIBDIR"
 
 if expr "$OCAMLVERSION" '>=' '4' > /dev/null ; then
    ODOC_DEF="-D OCAML_400"
